flink-connector/src/test/java/com/datastrato/gravitino/flink/connector/integration/test/hive/FlinkHiveCatalogIT.java

```diff
@@ -281,6 +291,59 @@ public void testGetCatalogFromGravitino() {
         numCatalogs, tableEnv.listCatalogs().length, "The created catalog should be dropped.");
   }
 
+  @Test
+  public void testHivePartitionTable() {
+    String databaseName = "test_create_hive_partition_table_db";
+    String tableName = "test_create_hive_partition_table";
+    String comment = "test comment";
+    String key = "test key";
+    String value = "test value";
+
+    doWithSchema(
+        currentCatalog(),
+        databaseName,
+        catalog -> {
+          TableResult result =
+              sql(
+                  "CREATE TABLE %s "
+                      + "(string_type STRING COMMENT 'string_type', "
+                      + " double_type DOUBLE COMMENT 'double_type')"
+                      + " PARTITIONED BY (string_type, double_type)"
+                      + " COMMENT '%s' WITH ("
+                      + "'%s' = '%s')",
+                  tableName, comment, key, value);
+          TestUtils.assertTableResult(result, ResultKind.SUCCESS);
+
+          Table table =
+              catalog.asTableCatalog().loadTable(NameIdentifier.of(databaseName, tableName));
+          Assertions.assertNotNull(table);
+          Assertions.assertEquals(comment, table.comment());
+          Assertions.assertEquals(value, table.properties().get(key));
+          Column[] columns =
+              new Column[] {
+                Column.of("string_type", Types.StringType.get(), "string_type", true, false, null),
+                Column.of("double_type", Types.DoubleType.get(), "double_type")
+              };
+          assertColumns(columns, table.columns());
+          Transform[] partitions =
+              new Transform[] {
+                Transforms.identity("string_type"), Transforms.identity("double_type")
+              };
+          Assertions.assertArrayEquals(partitions, table.partitioning());
+
+          TestUtils.assertTableResult(
```

Comment:
could you check the partition dir exists?

Reply:
Done